fee/src/lib.rs

```diff
@@ -82,11 +67,11 @@ fn calculate_signature_fee(
     signature_count.saturating_mul(lamports_per_signature)
 }
 
-struct SignatureCounts {
-    pub num_transaction_signatures: u64,
-    pub num_ed25519_signatures: u64,
-    pub num_secp256k1_signatures: u64,
-    pub num_secp256r1_signatures: u64,
+pub struct SignatureCounts {
+    num_transaction_signatures: u64,
+    num_ed25519_signatures: u64,
+    num_secp256k1_signatures: u64,
+    num_secp256r1_signatures: u64,
```

Comment:
we want to control conversions to this type inside this library that fee team owns to make sure nothing creeps in.